src/main/java/org/myrobotlab/service/LocalSpeech.java

```diff
@@ -142,6 +142,22 @@ protected void loadVoices() {
       addVoice("Linus", "male", "en-US", "festival");
     }
   }
+  
+  /**
+   * override default tts.exe path
+   * 
+   * @param ttsPath
+   *          - full path to windows tts.exe executable
+   * TODO - override also other os
+   */
+  public void setPath(String ttsPath) {

```

Comment:
This should be name  setTtsPath and getTtsPath to follow the java convention.  Setters and getters are named for the variable that they set and get.